dpu.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: config.openshift.io/v1
```

Comment:
do we want this file or not? it's my own for locally testing, as there is no CR file on repo

Reply:
I didn't commit mine but we can add it in an examples diectory

Reply:
ok I will move it. 
I will also create a "host" one too